ui/apps/platform/src/Components/ScannerV4IntegrationBanner.tsx

```diff
@@ -27,8 +27,8 @@ function ScannerV4IntegrationBanner() {
 
     const brandedText =
         branding.type === 'RHACS_BRANDING'
-            ? 'New Scanner V4 now generally available in RHACS 4.4.'
-            : 'New Scanner V4 now generally available in StackRox 4.4.';
```

Comment:
Is the deletion of **in** for StackRox by intention?

Reply:
Good catch! It is not.